flac/VITABUILD

```diff
@@ -0,0 +1,24 @@
+pkgname=flac
+pkgver=1.3.2
+pkgrel=1
+url="https://xiph.org/"
+source=("http://downloads.xiph.org/releases/flac/flac-${pkgver}.tar.xz")
+sha256sums=('91cfc3ed61dc40f47f050a109b08610667d73477af6ef36dcad31c31a4a8d53f')
+
+build() {
+  cd $pkgname-$pkgver
+  # fix memory.h inclusion
+  sed -ie 's/#include <memory.h>/\/\/#include <memory.h>/g' src/libFLAC/cpu.c
```

Comment:
Better use a [patch](https://github.com/vitasdk/packages/blob/master/libmad/libmad-0.15.1b-Provide-Thumb-2-alternative-code-for-MAD_F_MLN.patch) but that should do.